### ISSUE TITLE
Write consoleAlerts to stderr

### ISSUE THREAD

**Issue by rockymeza** [getsentry/raven-node#454]
**Do you want to request a *feature* or report a *bug*?**
feature

**Has someone had this problem before?**
no

**What is the current behavior?**
When Raven is configured with an empty DSN (aka in development), it outputs this to stdout:

    raven@2.4.2 alert: no DSN provided, error reporting disabled

this is very nice and I appreciate it. However when writing scripts that are supposed to write to stdout, this will mix into the output of the script, making it very difficult to use.

**What is the desired behavior?**

Would it be possible to write to stderr for these messages instead of stdout? This could come in the form of just using `console.warn` or some sort of utility to pass in a Console object that was configured differently, like this:

```javascript
const { Console } = require('console');

// somehow pass newConsole to Raven...
// This would not be as easy to use, but I suppose it provides some flexibility.
const newConsole = new Console({ stdout: process.stderr, stderr: process.stderr });
```

I am running raven@2.4.2, but I believe this behavior is still present in master (2.6-ish?).

https://github.com/getsentry/raven-node/blob/381dd6fe69f6c193e8f0fcaf46919efb68dc6e16/lib/utils.js#L126-L137

**Comment by rockymeza** [comment]
Thanks @kamilogorek !

**Comment by kamilogorek** [comment]
@rockymeza released as `2.6.1`
